hr_expense_sequence/tests/test_hr_expense_sequence.py

```diff
@@ -0,0 +1,43 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo.tests.common import SavepointCase
+
+
+class TestHrExpenseSequence(SavepointCase):
+
+    @classmethod
+    def setUpClass(self):
+        super().setUpClass()
+        self.expense_model = self.env['hr.expense']
+        self.expense_sheet_model = self.env['hr.expense.sheet']
+        self.product = self.env.ref("product.product_product_4")
+
+        employee_home = self.env['res.partner'].create({
+            'name': 'Employee Home Address',
+        })
+        self.employee = self.env['hr.employee'].create({
+            'name': 'Employee',
+            'address_home_id': employee_home.id,
+        })
+        self.expense = self.create_expense(self, 'Expense Test')
+        self.sheet = self.expense_sheet_model.create({
+            'name': 'Report Expense',
+            'employee_id': self.employee.id,
+        })
+
+    def create_expense(self, name):
+        """ Returns an open expense """
+        expense = self.expense_model.create({
+            'name': name,
+            'employee_id': self.employee.id,
+            'product_id': self.product.id,
+            'unit_amount': self.product.standard_price,
+            'quantity': 1,
+        })
+        expense.action_submit_expenses()
+        return expense
+
+    def test_create_sequence(self):
+        self.sheet.expense_line_ids = [
+            (6, 0, [self.expense.id])]
+        self.sheet.action_submit_sheet()
```

Comment:
When doing test, you need to self.assertXXX to make sure that the result is as expected.